utils/strings/strings.go

```diff
@@ -184,6 +184,19 @@ func ConvertStringSliceToMap(stringSlice []string) (stringInterfaceMap map[strin
 	return
 }
 
+func ParseEnvs(envs []string) map[string]string {
```

Comment:
i thought , we only need to optimize  this func `ConvertStringSliceToMap ` will meet request.